eslint/README.md

```diff
@@ -0,0 +1,18 @@
+## ESLint
+
+This directory contains ESLint configs that should be extended by every cloudify-ui project/module.
+
+### Usage
+
+Once `cloudify-ui-common` is installed as a dependency it is required to modify project's ESLint config (most probably `.eslintrc` or `.eslintrc.json` file) by adding the following entry (assuming ESLint config file is located in top level directory and it is a non-react project):
+```json
+{
+  "extends": ["./node_modules/cloudify-ui-common/eslint/eslint-common.json"]
```

Comment:
In the future we can have specific npm projects for those 2 configs.
Then they could be installed directly as dependency. That will reduce dependencies pollution.
For now - as we are not maintaining huge list of projects in UI - let's have it like that.